src/components/SignUp.js

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+
+export default function SignIn() {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    console.log({
+      name: data.get("name"),
+      email: data.get("email"),
+      password: data.get("password"),
+    });
+  };
+
+  return (
```

Comment:
Sign Up page should have a “Continue with GOOGLE” button, as we discussed in the last meeting. 
“Already have an account? Login”
Because these two pages were assigned to you, could you please fix the code and wireframe also? Thank you

Reply:
As we need only the sign-up button, for now, to connect with the back end, I am approving this PR.